services/movie_session.py

```diff
@@ -0,0 +1,58 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    movie = Movie.objects.get(id=movie_id)
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
```

Comment:
you don't need to perform these queries. You can set `cinema_hall` and `movie` by `id`s and not only objects